server/monitor.go

```diff
@@ -922,9 +922,9 @@ type LeafNodeOptsVarz struct {
 
 // RemoteLeafOptsVarz contains monitoring remote leaf node information
 type RemoteLeafOptsVarz struct {
-	LocalAccount string  `json:"local_account,omitempty"`
-	URL          string  `json:"url,omitempty"`
-	TLSTimeout   float64 `json:"tls_timeout,omitempty"`
+	LocalAccount string   `json:"local_account,omitempty"`
+	TLSTimeout   float64  `json:"tls_timeout,omitempty"`
+	URLs         []string `json:"urls,omitempty"`
```

Comment:
Humm... this is user visible and may break user apps since URL now is removed..

Reply:
Yes considered that but only user AFAIK is @ripienaar and I cleared with him.

Reply:
Ok then.